packages/slice-machine/lib/builders/SliceBuilder/Header/SimulatorButton/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect } from "react";
+import { Box, Flex, Close, Text, Link, useThemeUI } from "theme-ui";
+import { Button } from "@components/Button";
+
+import { BsPlayCircle } from "react-icons/bs";
+import { SIMULATOR_WINDOW_ID } from "@lib/consts";
+import { useRouter } from "next/router";
+import ReactTooltip from "react-tooltip";
+import { Frameworks } from "@slicemachine/core/build/models";
+
+import style from "./style.module.css";
+import { userHasSeenSimulatorToolTip } from "@src/modules/userContext";
+import { useSelector } from "react-redux";
+import { SliceMachineStoreType } from "@src/redux/type";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+import { getLinkToStorybookDocs } from "@src/modules/environment";
+
+const SimulatorButton: React.FC<{
+  framework: Frameworks;
+  isSimulatorAvailableForFramework: boolean;
+}> = ({ framework, isSimulatorAvailableForFramework }) => {
+  const router = useRouter();
+  const { theme } = useThemeUI();
+
+  const ref = React.createRef<HTMLButtonElement>();
+
+  const { setSeenSimulatorToolTip } = useSliceMachineActions();
+
+  const { hasSeenSimulatorTooTip, linkToStorybookDocs } = useSelector(
+    (store: SliceMachineStoreType) => ({
+      hasSeenSimulatorTooTip: userHasSeenSimulatorToolTip(store),
+      linkToStorybookDocs: getLinkToStorybookDocs(store),
+    })
+  );
+
+  useEffect(() => {
+    if (
+      ref.current &&
+      isSimulatorAvailableForFramework &&
+      !hasSeenSimulatorTooTip
+    ) {
+      const { current } = ref;
+      setTimeout(() => ReactTooltip.show(current), 5000);
+    }
+  }, []);
```

Comment:
eslint-react-hooks plugin will ask about deps here :)

Reply:
oooh, actually, a `useCallback` would be perfect here (because I don't want to run it on all ref updates.

```
const setRef = useCallback(node => {
    if (ref.current) {
        return;
    }
    
    if (node) {
        setTimeout(() => ReactTooltip.show(node), 5000);
    }
    ref.current = node
  }, [])
`